bpo-46658: shutil Lib enabling sendfile on solaris systems as it supports (#31165)
python/cpython: Misc/NEWS.d/next/Library/2022-02-06-15-53-30.bpo-46658.oRp0C-.rst

> Enabling ``os.sendfile`` on shlib on Solaris based systems.

Reviewer: ```suggestion
Enable use of :func:`os.sendfile` in :mod:`shutil` on Solaris.
```
